splitio/models/grammar/matchers/semver.py

```diff
@@ -277,3 +300,49 @@ def __str__(self):
     def _add_matcher_specific_properties_to_json(self):
         """Add matcher specific properties to base dict before returning it."""
         return {'matcherType': 'LESS_THAN_OR_EQUAL_TO_SEMVER', 'stringMatcherData': self._data}
+
+class BetweenSemverMatcher(Matcher):
+    """A matcher for Semver between."""
+
+    def _build(self, raw_matcher):
+        """
+        Build a BetweenSemverMatcher.
+
+        :param raw_matcher: raw matcher as fetched from splitChanges response.
+        :type raw_matcher: dict
+        """
+        self._data = raw_matcher.get('betweenStringMatcherData')
+        self._semver_start = Semver(self._data['start']) if self._data.get('start') is not None else None
```

Comment:
Should it be 
```
        self._semver_start = Semver.build(self._data['start'])
        self._semver_end = Semver.build(self._data['end'])
```
to avoid throwing an exception due to invalid semver string?

Reply:
fixed